contxt/cli/commands/iot.py

```diff
@@ -48,6 +47,33 @@ def groupings(clients: Clients, facility_id: int, fields: List[str], sort: str)
     print_table(items=items, keys=fields, sort_by=sort)
 
 
+@iot.command()
+@click.option("--output_id", required=True, type=int, help="Output ID to delete from")
+@click.option(
+    "--field", required=True, type=str, help="The field to delete from. Ex: power.demand, power.usage",
+)
+@click.option(
+    "--interval",
+    required=True,
+    type=click.Choice([str(v.value) for v in Window]),
+    callback=lambda ctx, param, value: Window(int(value)) if value is not None else None,
+    help="Time interval",
+)
+@click.option(
+    "--time",
+    required=True,
+    type=click.DateTime(),
+    help="The timestamp of the datapoint. Ex: 2020-07-21 05:31:00",
```

Comment:
how are we handling timezones here? your example is currently valid, but should it be? it's ambiguous what timezone you mean. local? utc?

we probably want to enforce that the timezone is specified

Reply:
Currently the exact timestamp of the outlier is provided by a manually run script, but I'll add a note that it's UTC